src/main/resources/db/migration/V28__seed_card_emotion_prompt.sql

```
@@ -0,0 +1,25 @@
+-- 카드 감정 분류(#134) 프롬프트를 시딩한다. 클라이언트가 emotion 없이 카드 생성을 요청하면
+-- 서버가 유저 메시지만 보고 대표 감정 하나를 고르는 데 쓴다. 분류 작업이라 COMMON과 조립하지
+-- 않는 단독 프롬프트다(EONGTTUNG_TOPIC과 같은 예외 - SystemPromptResolver 참고).
+-- 이미 값이 있으면 건드리지 않는다.
+-- model은 COMMON 행만 읽히는 자리표시자라 상수 대신 그 시점의 COMMON 값을 참조한다(V23 주석 참고).
+INSERT INTO llm_settings (prompt_type, model, system_prompt, updated_at)
+SELECT 'CARD_EMOTION', (SELECT s2.model FROM (SELECT model FROM llm_settings WHERE prompt_type = 'COMMON') s2), '[이번 타입: 카드 감정 분류]
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)